src/components/Shared/MultiSelect/index.scss

```diff
@@ -0,0 +1,29 @@
+@import "../../../styles/constants.scss";
+
+.multi-select {
+  margin-bottom: 24px;
+  > div {
+    > button {
+      width: 100%;
+      height: 40px;
+      margin: 0 !important;
```

Comment:
Why so many uses of `important!`? it should be a last resort and should not be used frequently.

Reply:
@mwogates please address this issue

Reply:
@roienatan @leviyehonatan, I fixed this. And we need some important style in this file, because we used React multi select node module and needed to customize its style. I fixed important styles in the other files too.